falcon/asgi/app.py

```diff
@@ -51,17 +51,27 @@
 # TODO(kgriffs): Rename the WSGI class to App with an API alias kept for
```

Comment:
Now that https://github.com/falconry/falcon/issues/1579 is closed, we could revise this TODO :arrow_up:

Reply:
done